runtime/v2/shim/shim.go

```diff
@@ -99,6 +101,7 @@ const (
 
 func parseFlags() {
 	flag.BoolVar(&debugFlag, "debug", false, "enable debug output in logs")
+	flag.BoolVar(&versionFlag, "v", false, "show the shim version and exit")
```

Comment:
Why not `-version`?

Reply:
s/"shim version"/"containerd version"/, because "show the shim version" flag is more likely to be expected print "2" for v2 shims.

Also, can we have the flag for `containerd-shim` (v1) as well?

Reply:
no, i'm not updating v1 shims anymore